lib/actions-oclif/env/add.ts

```diff
@@ -71,69 +73,55 @@ export default class EnvAddCmd extends Command {
 		'env add ' + new CommandHelp({ args: EnvAddCmd.args }).defaultUsage();
 
 	public static flags: flags.Input<FlagsDef> = {
-		application: flags.string({
-			char: 'a',
-			description: 'application name',
-			exclusive: ['device'],
-		}),
-		device: flags.string({
-			char: 'd',
-			description: 'device UUID',
-			exclusive: ['application'],
-		}),
-		help: flags.help({ char: 'h' }),
-		quiet: flags.boolean({
-			char: 'q',
-			description: 'suppress warning messages',
-			default: false,
-		}),
+		application: _.assign({ exclusive: ['device'] }, cf.application),
+		device: _.assign({ exclusive: ['application'] }, cf.device),
+		help: cf.help,
+		quiet: cf.quiet,
 	};
 
 	public async run() {
 		const { args: params, flags: options } = this.parse<FlagsDef, ArgsDef>(
 			EnvAddCmd,
 		);
-		const Bluebird = await import('bluebird');
-		const _ = await import('lodash');
+		const cmd = this;
 		const balena = (await import('balena-sdk')).fromSharedOptions();
 		const { exitWithExpectedError } = await import('../../utils/patterns');
 
-		const cmd = this;
+		if (params.value == null) {
+			params.value = process.env[params.name];
 
-		await Bluebird.try(async function() {
```

Comment:
@pdcastro For ease of review, this kind of refactor should be in a separate commit, or even better, a separate PR

Reply:
Done - I've split the changes to `add.ts` and `rm.ts` to a separate commit. I don't like multiple PRs for such simple refactoring because:
* Each PR creates a new CLI version and release, and CLI users and product/commercial folks (Shaun, Alex...) will be looking for the "value" of each new release.
* Users will be advised of the new CLI version available for download.
* Each PR/release results in hundreds of megabytes of assets created, uploaded and stored to the GitHub releases page.
* The CLI's automated testing still has significant room for improvement (as you know all too well, #1399), :-) and when bugs are reported, users and developers find themselves manually testing different versions to isolate the problem. The fewer "superfluous" releases, the better.